kuma/static/styles/components/payments/_page.scss

```diff
@@ -117,6 +119,10 @@
                 padding-left: 1em;
```

Comment:
I would suggest that we add `margin-top: 20px;` here to create a bit of space between the paragraph and the list.

Reply:
Changed the `p:last-of-type` rule to `p:last-child` rule instead, so we still have the margin between list and paragraph. Thank you for pointing this out!